test/active_model_serializers/test/serializer_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
+require 'rails-controller-testing' if Rails::VERSION::MAJOR >= 5
```

Comment:
I can look at that in a followup PR if you'd prefer. I think that'd be nicer at this point :)